mkdocs.yml

```diff
@@ -75,8 +75,29 @@ nav:
       - Core: reference/core.md
       - REST Apis: reference/rest_apis.md
       - Experimental:
-          - Object-Orientated Models: reference/oop/models.md
-          - Async Object-Orientated Models: reference/oop/models_async.md
+          - Mixins:
+              - AccessControllable: reference/experimental/mixins/access_controllable.md
+              - StorableContainer: reference/experimental/mixins/storable_container.md
+              - AsynchronousCommunicator: reference/experimental/mixins/asynchronous_communicator.md
+              - FailureStrategy: reference/experimental/mixins/failure_strategy.md
+          - Synchronous:
+              - Agent: reference/experimental/sync/agent.md
+              - Project: reference/experimental/sync/project.md
+              - Folder: reference/experimental/sync/folder.md
+              - File: reference/experimental/sync/file.md
+              - Table: reference/experimental/sync/table.md
+              - Activity: reference/experimental/sync/activity.md
+              - Team: reference/experimental/sync/team.md
+              - UserProfile: reference/experimental/sync/user_profile.md
+          - Asynchronous:
+              - Agent: reference/experimental/async/agent.md
+              - Project: reference/experimental/async/project.md
+              - Folder: reference/experimental/async/folder.md
+              - File: reference/experimental/async/file.md
+              - Table: reference/experimental/async/table.md
+              - Activity: reference/experimental/async/activity.md
+              - Team: reference/experimental/async/team.md
+              - UserProfile: reference/experimental/async/user_profile.md
```

Comment:
What do think about putting `Mixins` as the last in the list?

I was also thinking about 1 additional change to drop the additional navigation into the additional `Synchronous` directory. The purpose would be so that all the object models aren't hidden when scrolling down the page.

```
          - Agent: reference/experimental/sync/agent.md
          - ...
          - ...
          - Asynchronous:
              - Agent: reference/experimental/async/agent.md
              - ...
```
Current page:
![image](https://github.com/user-attachments/assets/c69a5af9-8b99-4700-8c91-4ba6cf2544a3)